lib/routes/komiic/comic.ts

```diff
@@ -0,0 +1,87 @@
+import { Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import got from '@/utils/got';
+
+export const route: Route = {
+    path: '/comic/:id/:chapterCnt?',
```

Comment:
Redundant parameter `chapterCnt`. The common parameter `limit` can achieve the same thing.

Reply:
In fact, I wrote this route based on [manhuagui/comic.ts](https://github.com/DIYgod/RSSHub/blob/master/lib/routes/manhuagui/comic.ts), so this parameter `chapterCnt` also comes from there.